### DISCUSSION TITLE
HEATMAP, GPU MODE, AMBIENT MODE NOT WORKING + KEYBOARD LIGHTS WON'T TURN OFF IN SLEEP MODE

### DISCUSSION THREAD

**Discussion by RAGE-HERO7** [seerge/g-helper#1776] [General]
Discribe the 2 problems

1st - after updating to ver 0.140.0 Three lighting animations on keyboard and lightbar (Ambient,gpu mode, heatmap) are not working :(

![20231223_132058](https://github.com/seerge/g-helper/assets/154586512/c59458d7-cdab-468b-ae1e-01371e717430)
![20231223_132257](https://github.com/seerge/g-helper/assets/154586512/3d86f63f-0a23-4b49-9b43-ad6bfaa256e4)
![20231223_132328](https://github.com/seerge/g-helper/assets/154586512/808c146d-c9ae-464b-8855-4f6c99648f2b)

2nd - After putting the laptop to sleep mode the light on the keyboard and lightbar won't turn off (both in plugged and battery), only screen goes off :(

https://github.com/seerge/g-helper/assets/154586512/bf25a00a-101c-4a1c-bf94-3c309f5359a8

LOG FILE
[log.txt](https://github.com/seerge/g-helper/files/13758600/log.txt)

Device - Asus Rog Strix G15 2021
Model No - (HQ403TS-G513QM)

Op System - Windows 11 22H2 Enterprise Ed

Spec - Amd Ryzen 9 5900HX, Nvidia Rtx 3060, 16gb ddr4 kingston Ram, 1tb Intel 670p ssd

*GHELPER VER 0.140.0
*No Armoury crate,my asus installed,
*All asus service's are stopped 
*All device drivers and Bios (ver.331) are uptodate

@seerge 🙏

**Comment by seerge** [comment]
@RAGE-HERO7 there is https://github.com/seerge/g-helper/releases/tag/v0.141 where it's working :)

**Reply by RAGE-HERO7** [reply]
Thank you 🫡
